edx_proctoring/views.py

```diff
@@ -852,6 +862,15 @@ def make_review(self, attempt, data, backend=None):
         except (User.DoesNotExist, KeyError):
             review.reviewed_by = None
 
+        # If the reviewing user is a user in the system (user may be None for automated reviews) and does
+        # not have permission to submit a review, log a warning.
+        course_id = attempt['proctored_exam']['course_id']
+        if review.reviewed_by is not None and not is_user_course_or_global_staff(review.reviewed_by, course_id):
+            LOG.warning(
```

Comment:
How about a test?

And I think the preferred way of formatting log strings (according to pylint?) is to have the logger do the format. So, `LOG.warning(msg, {'user': user, 'attempt': attempt_code})`

Reply:
For future reference: https://stackoverflow.com/questions/34619790/pylint-message-logging-format-interpolation.